parseutils.c

```diff
@@ -560,7 +565,17 @@ get_int64_from_file(char *ftr)
 		result = PG_INT64_MAX;
 	else
 	{
+        #if PG_VERSION_NUM < 150000
```

Comment:
I'd unindent all of these inline preprocessor macros; at least that's how I've seen this sort of version-specific code. (Again, project practice here.)  As currently written, it's a little hard to tease out the logic flow.

Reply:
Didn't even notice that for the other one. Thanks. Fixed for consistency.